enterprise/views.py

```diff
@@ -637,7 +637,7 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             ),
             'course_modes': filter_audit_course_modes(enterprise_customer, course_modes),
             'course_effort': course_effort,
-            'course_full_description': clean_html_for_template_rendering(course_run['full_description']),
+            'course_full_description': clean_html_for_template_rendering(course_run['full_description'] or ''),
```

Comment:
@douglashall the full description coming back for all of the georgia tech courses had fake new lines in between tags (ie literal characters \ and n, not an actual newline). so after rendering it as html in the template, i was seeing '\n' in the course modal. i didn't dig into how that data was coming back from discovery with this weird formatting, and i'm not sure if it's something that happens across courses.